app/controllers/admin/members_controller.rb

```diff
@@ -35,4 +39,47 @@ def permitted_params
     params.require(:member).permit %i(email username approved first_name
     last_name phone postal_street postal_zip postal_city)
   end
+
+  def load_website
+    if current_user.admin?
+      @website = current_website
+    else
+      @website = current_user.websites.find(current_website.id)
+    end
+  end
+
+  def unset_blank_params
+    %i(email username approved name phone postal).each do |attr|
```

Comment:
no, but it might be totally unnecessary since we're also doing
```ruby
p = params.permit(%i(email username approved name phone postal)).
  reject { |_key, val| val.blank? }
```